src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -77,6 +77,14 @@
             doc = "One or more input files (or lists of files) to compare fingerprints for.")
     public List<File> INPUT;
 
+    @Argument(shortName = "I2", optional = true,
+            doc = "One or more input files (or lists of files) to compare fingerprints for. If this option is given " +
+                    "the program to compare each sample in INPUT, with the sample from " +
+                    "INPUT2 that have the same sample ID. When operating in this mode, each sample in " +
+                    "INPUT must also have a corresponding sample in INPUT2. If this is violated, the program will proceed to check the matching samples, but report " +
+                    "the missing samples and return a non-zero error-code. In addition, data will be grouped by SAMPLE regardless of the value of CROSSCHECK_BY." )
```

Comment:
Is the "In addition" mean if the condition is violated or merely if INPUT2 is specified?

Reply:
done